app/models/health_care_application.rb

```diff
@@ -103,11 +106,9 @@ def process!
       raise(Common::Exceptions::ValidationErrors, self)
     end
 
-    has_email = parsed_form['email'].present?
-
-    if has_email || async_compatible
+    if email? || async_compatible
```

Comment:
Used the new new `email?` method here to DRY it up and simplify what we have to pass in.